internal/batches/ui/task_exec_tui_test.go

```diff
@@ -205,6 +197,70 @@ func TestTaskExecTUI_Integration(t *testing.T) {
 	})
 }
 
+func TestProgressUpdateAfterComplete(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Something emits different escape codes on windows.")
```

Comment:
Of course 😆

Reply:
In fairness, I copy/pasted that from the other integration test — but only after it failed on Windows. :window: